DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Services;
+using Google.Apis.Sheets.v4;
+using Google.Apis.Sheets.v4.Data;
+using static Google.Apis.Sheets.v4.SpreadsheetsResource.ValuesResource;
+
+namespace DiscordBot.DataAccess
+{
+    public class EventsSheetsService
+    {
+        private static readonly string[] scopes = { SheetsService.Scope.Spreadsheets };
+        private static readonly string applicationName = "Softwire Discord Bot";
+        private static readonly string spreadsheetId = "";
```

Comment:
I assume this isn't meant to be empty string, is this the bit you haven't set up to be stored securely yet? If so I'd recommend you get the value from the configuration object. You can set it using [secret manager](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.1&tabs=windows#secret-manager) for local development and using [environment variables](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1#environment-variables) when deployed 🙂

Reply:
Yeah this is the bit I haven't set up secure storage for. Shiv is trying to get it all working with the Discord secrets (as part of him getting CI/CD working), so I was planning to copy what he did for these secrets once he had got it all working, if that's ok? 🙂

Reply:
In my local code I have used the visual studio build in environment variables thing where it adds them to a launchSettings.json and you can use `Environment.GetEnvironmentVariable()` to access it.

Reply:
@benjimarshall you will only be able to use the secrets to set the environment variables as you can't access the secrets in the code. And that won't work for setting it locally. So I'd still recommend environment variables and secret manager. But fine to do in a separate PR after Shiv gets his stuff working 🙂 

@SChandarana you can use launchSettings.json but then you need to make sure you don't commit that file which is a bit annoying. So secret manager might be easier 🙂 I haven't used env variables with azure deployment before, but let me know if you're having problems with it as I'm happy to have a look 🙂